src/controller/response/update-key-response.ts

```diff
@@ -18,6 +18,12 @@
  *  @license
  */
 
+/**
+ * This is the module page of the update-key-response.
+ *
+ * @module internal/controllers
```

Comment:
If you would have checked the contents of this file, you would have seen this response is used only in the `user-controller` when requesting a new API key. So either `users` or `authentication`.